mismi-sqs/test/Test/Mismi/SQS.hs

```diff
@@ -55,9 +57,10 @@ runSQSWithQueueArg v (Queue qn r) f =
 
 runSQSWithCfgWithDefaults :: AWS b -> IO b
 runSQSWithCfgWithDefaults f = do
-  r <- getRegionFromEnv
-  let e = fromMaybe Sydney $ P.rightToMaybe r
-  runSQSWithRegion e f
+  r <- getRegionFromEnv `catch` \case
+     RegionMissing -> return Sydney
```

Comment:
This is definitely one of those cases where `catch` is being used as control flow - `getRegionFromEnv` should just return `Maybe Region` or thereabouts